src/Confluent.Kafka/Consumer.cs

```diff
@@ -414,6 +414,25 @@ public void Unassign()
             => consumer.Unassign();
 
         /// <summary>
+        ///     Offset is stored for commit based on the topic/partition/offset of a message.
+        ///     The next message to be read will be that following <paramref name="message" />.
+        /// </summary>
+        /// <param name="message">
+        ///     The message used to determine the stored offset.
+        /// </param>
+        /// <remarks>
+        ///     A consumer which has position N has consumed records with offsets 0 
```

Comment:
This is incorrect, the consumer can start consuming at any offset.
I think this remark needs a some tweaking, it reads a bit hard.

Also make sure to pick up the information from here:
https://github.com/edenhill/librdkafka/blob/master/src/rdkafka.h#L2360

Reply:
I will update the docs